aksenov/VMindbooke.Bot/Application/UserService.cs

```diff
@@ -0,0 +1,28 @@
+﻿using VMindbooke.Bot.Domain;
+
+namespace VMindbooke.Bot.Application
+{
+    public class UserService
+    {
+        private APIRequestsService _apiService;
+
+        public UserService(APIRequestsService apiService)
+        {
+            _apiService = apiService;
+        }
+
+        public bool DoesUserExists(int userId)
+        {
+            var user = _apiService.GetUser(userId);
+            if (user != null && user.IsValid())
+                return true;
+
+            return false;
```

Comment:
```suggestion
return user != null && user.IsValid();
```
тебе ж прям райдер подсказывает